sdk/devcenter/developer-devcenter-rest/README.md

```diff
@@ -0,0 +1,59 @@
+# AzureDevCenter client library for JavaScript
+
+Azure DevCenter Client
```

Comment:
One suggestion: I would recommend you to add more descriptions for your service and REST client [[example link](https://raw.githubusercontent.com/Azure/azure-sdk-for-js/main/sdk/agrifood/agrifood-farming-rest/README.md)].